apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/RealCache.java

```diff
@@ -9,22 +15,35 @@
 public final class RealCache implements Cache {
   private final CacheStore cacheStore;
   private final CacheKeyResolver cacheKeyResolver;
+  private Map<RecordChangeSubscriber, Set<String>> subscribers;
```

Comment:
Should we make `RecordChangeSubscriber ` as weak ref? So WeakHashMap?
Then it will be user responsibility to keep around subscribtion?